src/main/java/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud.java

```diff
@@ -913,6 +914,11 @@ public String toString() {
     }
 
     private Object readResolve() {
+        if ((serverCertificate != null) && !serverCertificate.startsWith("-----BEGIN CERTIFICATE-----")) {
```

Comment:
What about certificates that happen to have a blank line before the `-----BEGIN CERTIFICATE-----`?